client/me/purchases/manage-purchase/notices.jsx

```diff
@@ -57,7 +59,7 @@ class PurchaseNotice extends Component {
 					{
 						args: {
 							purchaseName: getName( purchase ),
-							expiry: moment( purchase.expiryMoment ).fromNow(),
+							expiry: expiry.fromNow(),
```

Comment:
Here we call `fromNow()` on `false` if the expiry date is not valid. Again, the original code didn't check that either.

Reply:
Check removed.